src/client/v2/algod/models/base.ts

```diff
@@ -1,63 +1,71 @@
-import { Address } from '../../../../types/address';
-
 /**
  * Base class for models
  */
 
-/* eslint-disable no-underscore-dangle,camelcase,class-methods-use-this */
-export default class BaseModel {
-  attribute_map: Record<string, string>;
+/* eslint-disable no-underscore-dangle,camelcase */
```

Comment:
ESLint disable markers should be re-enabled after the disabled code block. I think these dangling `eslint-disables` are actually my fault, but we might as well add them now. Additionally, I'm not sure that disabling `no-redeclare` and `no-unused-vars` is required now that we're using the [`@typescript-eslint/parser` versions of those rules](https://github.com/algorand/js-algorand-sdk/blob/develop/.eslintrc.js#L31-L34).

Reply:
My latest commit re-enabled the rules. However, I found that my eslint still required them to be there.